extensions/cornerstone/src/init.js

```diff
@@ -106,6 +106,73 @@ export default function init({ servicesManager, configuration }) {
     tools.push(...toolsGroupedByType[toolsGroup])
   );
 
+  const mapCornerstoneAnnotationToMeasurementServiceFormat = eventData => {
+    const { toolType, element, measurementData } = eventData;
+
+    const validToolType = toolType => !['Length'].includes(toolType);
+    if (validToolType(toolType)) {
+      return { error: 'Invalid tool type' };
+    }
+
+    const enabledElement = cornerstone.getEnabledElement(element);
+    const imageId = enabledElement.image.imageId;
+    const sopInstance = cornerstone.metaData.get('instance', imageId);
+    const sopInstanceUid = sopInstance.sopInstanceUid;
+    const frameOfReferenceUid = sopInstance.frameOfReferenceUID;
+    const series = cornerstone.metaData.get('series', imageId);
+    const seriesInstanceUid = series.seriesInstanceUid;
+
+    const points = [];
+    points.push(measurementData.handles);
+
+    return {
+      imageId,
+      sopInstanceUid,
+      frameOfReferenceUID: frameOfReferenceUid,
+      referenceSeriesUID: seriesInstanceUid,
+      points, // points[] (x, y, z)
```

Comment:
We use [x1, y1, x2, y2] when we have the sopInstancrUid.
We use [x1, y1, z1, x2, y2, z2] if we only have a frameOfReferenceUid

Also, note difference in UID casing in object.

We wouldn't use imageId here. Link to annotation's imageIdSpecificToolstate would be via `measurementId` or `sopInstanceUid`. If there's no easy way to do this today, we can expand cornerstone tool's api

Reply:
This is where we would specify the `measurementId`. If we don't use something generated by cornerstone tools here, the alternative would be to have a way for us to `set` the ID for a given annotation (which we could do after `createOrUpdate` is called)